native/packages/react-native-bpk-component-button/readme.md

```diff
@@ -47,7 +45,7 @@ export default class App extends Component {
         <BpkButton disabled type="destructive" title={translationHelper.translate('BOOK_FLIGHT')} onPress={() => {}} />
         <BpkButton large type="primary" title={translationHelper.translate('BOOK_FLIGHT')} onPress={() => {}} />
 
-        <BpkButton type="featured" title={translationHelper.translate('BOOK_FLIGHT')} icon={ArrowImage} iconOnly onPress={() => {}} />
+        <BpkButton type="featured" title={translationHelper.translate('BOOK_FLIGHT')} icon="baggage" iconOnly onPress={() => {}} />
```

Comment:
It would still be good to have a constant for the icons so instead of `"baggage"` we could have `ICONS.BAGGAGE`

Reply:
Yeah that's a good point. I'm wondering if it should be a separate PR though as it's something `BpkIcon` would export and I don't want to conflate this PR though..

Reply:
It absolutely should be. Is there a jira card open for this?

Reply:
There is now: https://gojira.skyscanner.net/browse/BPK-1078